### ISSUE TITLE
Collapsing column breaks keyboard shortcuts

### ISSUE THREAD

**Issue by janpio** [ryz310/github_project_enhancer#19]
You can use <kbd>SPACE</kbd> to "select" a card in Github projects, then <kbd>Left</kbd> or <kbd>Right</kbd> to go through the columns, then place the card with <kbd>Enter</kbd> again to be moved to that column.

When using this extension to collapse a column, this doesn't seem to work any more unfortunately :(

**Comment by ryz310** [comment]
@janpio 

Oh, I've never known that operation.
You know, a chrome extension cannot meddle original javascript behavior so I don't know it can be fixed but I'll try investigate original implementation.

**Comment by janpio** [comment]
Yeah, I was hoping when collapsed the cards would just move into the same location, but invisible, but somehow they seem to get "lost" and then not appear in the not collapsed column after you moved them far enough. Maybe you can make your changes work with that implementation somehow.



> Oh, I've never known that operation.

Yeah on big boards it becomes more relevant, as moving cards with your mouse becomes laggy and error prone. Also moving cards over many columns where you have to scroll to reach the target doesn't work very well, but using the keyboard it is quick and simple.
